tests/Aspire.Hosting.Azure.Tests/AzureSchemaTests.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Tests.Schema;
+using Xunit;
+
+namespace Aspire.Hosting.Azure.Tests;
+
+public class AzureSchemaTests
+{
+    [Fact]
+    public void ValidateApplicationSamples()
+    {
+        new SchemaTests().ValidateApplicationSamples("CdkResourceWithChildResource", (IDistributedApplicationBuilder builder) =>
```

Comment:
Why does this say `CdkResourceWithChildResource`? Will CDK be used here because of `PublishAsAzurePostgresFlexibleServer`?

And is `CDK` - AWS CDK being used for deploying to Azure here?

Reply:
This is just moving the existing test in:

https://github.com/dotnet/aspire/blob/df935f4ee2e76d088235ea0fe91a0b61405cb3fe/tests/Aspire.Hosting.Tests/Schema/SchemaTests.cs#L110-L114

That test needed to be moved because Aspire.Hosting.Tests is no longer referencing the Aspire.Hosting.Azure libraries. So I moved the test to the Azure tests.